src/Symfony/Component/Console/Event/ConsoleExceptionEvent.php

```diff
@@ -27,12 +25,21 @@ class ConsoleExceptionEvent extends ConsoleEvent
     private $exception;
     private $exitCode;
 
-    public function __construct(Command $command, InputInterface $input, OutputInterface $output, \Exception $exception, $exitCode, $deprecation = true)
+    public function __construct($input, $output, $exception, $exitCode, $command = null, $deprecation = true)
```

Comment:
Given that the class is deprecated anyway I would just allow `$command` to be `null` instead of shifting constructor arguments.